scripts/picard.in

```diff
@@ -1,3 +1,33 @@
 #!/usr/bin/env python3
-from picard.tagger import main
-main('%(localedir)s', %(autoupdate)s)
+
+try:
+    from picard.tagger import main
+    main('%(localedir)s', %(autoupdate)s)
+except SystemExit:
+    raise  # Just continue with a normal application exit
+except:  # noqa: F722
+    # First try to get traceback information and write it to a log file
+    # with minimum chance to fail.
+    import os
+    import sys
+    import tempfile
+    import traceback
+    trace = traceback.format_exc()
+    try:
+        (fd, logfile) = tempfile.mkstemp(".log", "picard-crash-")
+        os.write(fd, trace.encode(errors="replace"))
+        os.close(fd)
+    except:  # noqa: F722
+        print("Failed writing log file {0}".format(logfile), file=sys.stderr)
+        logfile = None
+
+    # Display the crash information to the user as a dialog. This requires
+    # importing Qt5 and has some potential to fail if things are broken.
+    from PyQt5 import QtWidgets
+    app = QtWidgets.QApplication(sys.argv)
+    msg = []
+    if logfile:
+        msg.append("A logfile has been written to {0}".format(logfile))
+    msg.append("Error details: \n{0}".format(trace))
+    QtWidgets.QMessageBox.critical(None, "Picard terminated unexpectedly", "\n\n".join(msg))
```

Comment:
It seems HTML-like tags are interpreted by `QtWidgets.QMessageBox.critical()`. It is an issue since we can't control what's in the trace text, it also concerns file path (while I doubt it can contain characters triggering RichText autodetection, better be safe).

Unfortunately, you can't change text format (and autodetection) when using static QMessageBox methods.

I did some experiments, and this trace is a good canditate for detailed text feature of QMessageBox, it is in text plain format by default:

Here is the test I did:

```python
import sys
from PyQt5.QtCore import Qt
from PyQt5.QtWidgets import QApplication, QMessageBox

app = QApplication(sys.argv)

msgbox = QMessageBox()
msgbox.setIcon(QMessageBox.Critical)
msgbox.setWindowTitle("Picard crashed")
msgbox.setTextFormat(Qt.PlainText)
msg = ["Picard <b>crashed</b>"]
logfile = '/x/<b>dddd</b>'
if logfile:
    msg.append("A logfile has been written to {0}".format(logfile))
msgbox.setText("\n\n".join(msg))
msgbox.setDetailedText("This is the trace<b>xxx<b>\nsfklslk")
msgbox.setStandardButtons(QMessageBox.Close)
msgbox.setDefaultButton(QMessageBox.Close)
msgbox.exec()
```
I intentionnaly used HTML-like tags in texts to test whether autodetection was on and if output was actually in plain text.
Here is the result on my system:
![image](https://user-images.githubusercontent.com/151042/74205086-162b2680-4c76-11ea-9909-7b924b575ad9.png)

Of course, it isn't as simple as original code, but it's the only way to prevent text format autodetection.